api/src/test/java/org/pmiops/workbench/access/UserServiceAccessTest.java

```diff
@@ -114,6 +123,7 @@ DbUser getDbUser() {
   @Before
   public void setUp() {
     providedWorkbenchConfig = WorkbenchConfig.createEmptyConfig();
+    providedWorkbenchConfig.accessRenewal.expiryDays = EXPIRATION_DAYS;
```

Comment:
we always set it to this, and this is safe to set regardless of the feature flag setting